packages/core/src/model.js

```diff
@@ -2312,7 +2314,7 @@ ${associationOwner._getAssociationDebugList()}`);
         }
       }
 
-      if (options.ignoreDuplicates && ['mssql', 'db2', 'ibmi'].includes(dialect)) {
+      if (options.ignoreDuplicates && ['mssql', 'db2', 'ibmi', 'oracle'].includes(dialect)) {
         throw new Error(`${dialect} does not support the ignoreDuplicates option.`);
       }
```

Comment:
In the fashion of the boy scout rule (removing technical debt when you touch it); can you make this a `dialect.supports` property instead? And even though `updateOnDuplicate` is not touched by oracle, I would not mind if you improved that one as well.